python/pysnaptest/snapshot.py

```diff
@@ -22,10 +22,10 @@ def extract_from_pytest_env(
 
 
 def assert_json_snapshot(
-    result: Any, snapshot_path: str | None = None, snapshot_name: str | None = None
+    result: Any, snapshot_path: str | None = None, snapshot_name: str | None = None, redactions: Optional[Dict[str, str]] = None
 ):
     test_info = extract_from_pytest_env(snapshot_path, snapshot_name)
-    _assert_json_snapshot(test_info, result)
+    _assert_json_snapshot(test_info, result, redactions)
 
 
 def assert_csv_snapshot(
```

Comment:
As an aside, I haven't actually figured out how to test this - I can't find any obvious python types that are serialisable as csvs. Probably something we should dig into.